orchestra/tests/test_task_lifecycle.py

```diff
@@ -498,7 +498,8 @@ def test_malformed_assignment_policy(self):
             slug='machine_step',
             is_human=False,
             assignment_policy={
-                'policy': 'previously_completed_steps',
+                'policy': ('orchestra.assignment_policies.'
```

Comment:
it would be:
```
"policy_function": {
            "module": "orchestra.assignment_policies.",
            "name": "previously_completed_steps"
        }
```

To mimick this:
 https://github.com/unlimitedlabs/orchestra/blob/assignment-policy-as-function/journalism_workflow/v1/version.json#L95

I actually like your implementation better, and pydoc.locate seems cleaner.  if you opt for your current version, consider 1) naming it `policy_function` and 2) adding a TODO to make machine functions use the same schema